android/app/google-services.json

```diff
@@ -0,0 +1,29 @@
+{
+  "project_info": {
+    "project_number": "752081205061",
```

Comment:
We would need to update README on how notification infra is setup and which keys needs to be changed and where.